<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSelectionWidgetClass</class>
 <widget class="QWidget" name="ColorSelectionWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="colorSelectionLabel">
     <property name="toolTip">
      <string>Color selection controls are all to the right.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../icons/ViewWidgetsIcons.qrc">:/VatesSimpleGuiViewWidgets/icons/pqEditColor24.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoColorScaleCheckBox">
     <property name="text">
      <string>Automatic Color Scaling</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useLogScaleCheckBox">
     <property name="text">
      <string>Use Log Scaling</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="minValLabel">
     <property name="text">
      <string>Minimum</string>
     </property>
     <property name="buddy">
      <cstring>minValLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="minValLineEdit">
     <property name="toolTip">
      <string>Set the minimum value for the color scale.</string>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="maxValLabel">
     <property name="text">
      <string>Maximum</string>
     </property>
     <property name="buddy">
      <cstring>maxValLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="maxValLineEdit">
     <property name="toolTip">
      <string>Set the maximum value for the color scale.</string>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="presetButton">
     <property name="text">
      <string>Color Preset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons/ViewWidgetsIcons.qrc"/>
 </resources>
 <connections/>
</ui>
